### PR TITLE
changes to log the tracer logs as JSON object

### PR DESCRIPTION
**Pull request by suryasoma**

### CODE REVIEW

File: bigquery-connector-common/src/main/java/com/google/cloud/bigquery/connector/common/LoggingBigQueryStorageReadRowsTracer.java [hunk 1]
```diff
@@ -116,16 +118,18 @@ private static long perSecond(DurationTimer timer, long metric) {
...
+    jsonObject.addProperty("Started", startTime == null ? "" : startTime.toString());
+    jsonObject.addProperty("Ended", endTime == null ? "" : endTime.toString());
+    jsonObject.addProperty("Parse Timings", format(parseTime));
+    jsonObject.addProperty("Time in Spark", difference(sparkTime, parseTime));
+    jsonObject.addProperty("Time waiting for service", format(serviceTime));
+    jsonObject.addProperty("Bytes/s", perSecond(serviceTime, bytes));
+    jsonObject.addProperty("Rows/s", perSecond(parseTime, rows));
+    jsonObject.addProperty("Bytes", bytes);
+    jsonObject.addProperty("Rows", rows);
+    log.info("Tracer Logs:{}", new Gson().toJson(jsonObject));
```
Comment (nileyadav) [inline bigquery-connector-common/src/main/java/com/google/cloud/bigquery/connector/common/LoggingBigQueryStorageReadRowsTracer.java:132]: Is similar change required for create read session log as well?
bigquery-connector-common/src/main/java/com/google/cloud/bigquery/connector/common/ReadSessionCreator.java : 150

Comment (suryasoma) [reply bigquery-connector-common/src/main/java/com/google/cloud/bigquery/connector/common/LoggingBigQueryStorageReadRowsTracer.java:132]: Yeah it is Nilesh, but this change is mainly to get total bytes read and records read. 
Will work on ReadSessionCreator (to get the scan time) soon and raise another PR for it.

**Review by davidrabinowitz — APPROVED** [review]

**Review by nileyadav — APPROVED** [review]

**Pull request merged**